### ISSUE TITLE
Setup script: figure out env var storage

### ISSUE THREAD

**Issue by ace-n** [GoogleCloudPlatform/emblem#231]
## TL;DR
Emblem uses several env vars; how should we provision them in Cloud Run?

## Context
To determine the env-vars in use, we looked at `config.py` and confirmed that no `getenv()` calls existed elsewhere in the codebase.

Based on [this guidance](https://github.com/GoogleCloudPlatform/emblem/blob/main/docs/config-secret.md), the only env-var value that can be added via Cloud Build is `EMBLEM_API_URL`.

All other values (i.e. those listed below) **must** be included as Cloud Run environment variables _using [Cloud Secret Manager](https://cloud.google.com/run/docs/configuring/secrets#command-line)_:

| Secret Name              | Env var name     |
| ------------------------ | ---------------- |
| `client-id-secret`      | `CLIENT_ID`      |
| `client-secret-secret`  | `CLIENT_SECRET`  |
| `redirect-uri-secret`   | `REDIRECT_URI`   |
| ~`website-session-secret`~ | ~`SESSION_SECRET`~ |

_Note: most of these values are related to OAuth 2.0 configuration._

## Unanswered questions
The main unanswered questions are:

> Can OAuth 2.0 be configured programmatically (i.e. [via `gcloud`](https://cloud.google.com/iap/docs/programmatic-oauth-clients#configuring_consent_screen))?
> Can OAuth 2.0 credentials for one GCP project be used in another?

The answers to these questions will affect how we decide to proceed. As such, we need to answer them before we can come up with an implementation plan.
